cmd/infracost/output.go

```diff
@@ -32,15 +32,11 @@ func outputCmd(cfg *config.Config) *cobra.Command {
 
   Merge multiple Infracost JSON files:
 
-      infracost output --format json --path out*.json`,
+			infracost output --format json --path out*.json`,
+		ValidArgs: []string{"--", "-"},
```

Comment:
What does this do?

Reply:
Because without that autocompletion with flags doesn't work. Maybe exist another the way to do it, but I didn't find it